<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerView</class>
 <widget class="QDialog" name="ManagerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>餐厅经理</string>
  </property>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>375</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbCook">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>75</y>
     <width>450</width>
     <height>325</height>
    </rect>
   </property>
   <property name="title">
    <string>厨师工作情况</string>
   </property>
   <widget class="QTableView" name="tbCook">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCook">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCook">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择一名厨师</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbWaitor">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>75</y>
     <width>450</width>
     <height>650</height>
    </rect>
   </property>
   <property name="title">
    <string>服务员工作情况</string>
   </property>
   <widget class="QComboBox" name="cbWaitor">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblWaitor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择一名服务员</string>
    </property>
   </widget>
   <widget class="QTableView" name="tbWaitor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tbComm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>350</width>
      <height>275</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblComm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>315</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>获得的评价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnProfile">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>25</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>个人资料</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSeatVacance">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>25</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>设置餐桌数</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbVolume">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>425</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string>菜品销量</string>
   </property>
   <widget class="QTableWidget" name="tbVolume">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>350</width>
      <height>225</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cbCook</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ManagerView</receiver>
   <slot>BindCook(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbWaitor</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ManagerView</receiver>
   <slot>BindWaitor(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>858</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProfile</sender>
   <signal>clicked()</signal>
   <receiver>ManagerView</receiver>
   <slot>OnProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSeatVacance</sender>
   <signal>clicked()</signal>
   <receiver>ManagerView</receiver>
   <slot>OnSetTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BindCook(QString)</slot>
  <slot>BindWaitor(QString)</slot>
  <slot>OnProfile()</slot>
  <slot>OnSetTable()</slot>
 </slots>
</ui>
